server/portal/apps/projects/management/commands/migrate-projects.py

```diff
@@ -17,16 +18,16 @@ class Command(BaseCommand):
     help = (
         'Reconcile projects from CEPv1 migration that do not have '
         'a PI assigned. This will attempt to assign the creator of '
-        'these projects as PI.'
+        'these projects as PI, as well as index all projects.'
     )
 
     def handle(self, *args, **options):
         """Handle command."""
         agc = service_account()
-        metadatas = ProjectMetadata.objects.all().filter(pi=None)
+        metadatas = ProjectMetadata.objects.all()
         for meta in metadatas:
+            index_project.apply_async(args=[meta.project_id])
             try:
```

Comment:
Now that we're looping over all metadata objects, should we check for pi here before entering the original try/except block?

Reply:
Yes, thanks for catching that ><